kvlds/dispatch_mr.c

```diff
@@ -524,7 +524,7 @@ callback_synced(void * cookie)
 			break;
 		case PROTO_KVLDS_CAD:
 			if (proto_kvlds_response_cad(B->WQ, R->ID,
-			    req->opdone))
+			    !req->opdone))
```

Comment:
I would spell these as `req->opdone ? 0 : 1` to make it clear that we want those two particular values and not merely logical negation.